pkg/os/os_windows.go

```diff
@@ -0,0 +1,124 @@
+// +build windows
```

Comment:
You don't need this line, because the file name has `_windows.go`

Reply:
Good catch